components/content/content-editor/content-editor.ts

```diff
@@ -339,6 +340,11 @@ export default class AppContentEditor extends Vue implements ContentOwner {
 		}
 	}
 
+	public updateIsEmpty(state: EditorState) {
+		// The "empty" prosemirror document takes up a length of 4.
+		this.isEmpty = state.doc.nodeSize <= 4;
```

Comment:
can a super short doc take up less than 4 somehow?

Reply:
No, 4 is the default size for no content at all (doc and paragraph opening and closing tags).
This is not the text length or even content length, it's prosemirror's calculated length for their internal doc thing.